cloudformation/database/serverless-database.yaml

```diff
@@ -237,106 +235,127 @@ Resources:
         - Key: 'parallelcluster:usecase'
           Value: 'slurm accounting'
       VpcId: !Ref Vpc
-  ServerlessDatabaseClusterSecurityGroupInboundRule:
+  AccountingClusterSecurityGroupInboundRule:
     Type: 'AWS::EC2::SecurityGroupIngress'
     Properties:
       IpProtocol: tcp
       Description: Allow incoming connections from client security group
       FromPort: !GetAtt
-        - ServerlessDatabaseCluster
+        - AccountingCluster
         - Endpoint.Port
       GroupId: !GetAtt
-        - ServerlessDatabaseClusterSecurityGroup
+        - AccountingClusterSecurityGroup
         - GroupId
       SourceSecurityGroupId: !GetAtt
-        - ServerlessClusterDatabaseClientSecurityGroup
+        - AccountingClusterClientSecurityGroup
         - GroupId
       ToPort: !GetAtt
-        - ServerlessDatabaseCluster
+        - AccountingCluster
         - Endpoint.Port
-  ServerlessDatabaseClusterAdminSecret:
+  AccountingClusterAdminSecret:
     Type: 'AWS::SecretsManager::Secret'
     Properties:
       Description: 'Serverless Database Cluster Administrator Password'
       SecretString: !Ref AdminPasswordSecretString
       Tags:
         - Key: 'parallelcluster:usecase'
           Value: 'slurm accounting'
-  ServerlessDatabaseCluster:
+  AccountingCluster:
     Type: 'AWS::RDS::DBCluster'
     Properties:
-      Engine: aurora-mysql
+      DBClusterIdentifier: !Ref ClusterName
+      Engine: "aurora-mysql"
+      EngineVersion: "8.0.mysql_aurora.3.02.1"
```

Comment:
(minor) is there a way to pick the latest one by any chance?

Reply:
I don't know? I can look into it - maybe a parameter type exists for selecting this value or maybe there is a built-in function (though I haven't come across it)